lib/pipely/build/template.rb

```diff
@@ -12,6 +12,8 @@ module Build
     class Template
       include TemplateHelpers
 
+      attr_accessor :pipeline_id
```

Comment:
I believe to make this work the way we want, we would have to then use another `template.apply_config` from the Definition to the Template in `to_json`.